decoder/vaapidecoder_base.h

```diff
@@ -85,6 +85,8 @@ class VaapiDecoderBase:public IVideoDecoder {
 
     //do not use this, we will remove this in near future
     virtual VADisplay getDisplayID();
+
+    virtual bool isSVCTFrame() { return true; }
```

Comment:
Different decoder may handle SVC in different way, this not needed in base class